### PR TITLE
Added my profile 

### PR DESCRIPTION
**Pull request by samyabrata-maji**

### CODE REVIEW

File: Readmes/SamyabrataMaji/README.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+<a src="https://sammaji.hashnode.dev" target="_blank" rel="noreferrer noopener"><img src="https://img.icons8.com/color/48/null/hashnode.png"/></a>&ensp;
+<a src="https://twitter.com/sammaji15" target="_blank" rel="noreferrer noopener"><img src="https://img.icons8.com/color/48/null/gmail--v1.png"/></a>&ensp;
+<a src="https://twitter.com/sammaji15" target="_blank" rel="noreferrer noopener"><img src="https://img.icons8.com/color/48/null/discord--v2.png"/></a>&ensp;
+<a src="https://twitter.com/sammaji15" target="_blank" rel="noreferrer noopener"><img src="https://img.icons8.com/fluency/48/null/instagram-new.png"/></a>&ensp;
+<a src="https://linkfree.eddiehub.io/samyabrata-maji" target="_blank" rel="noreferrer noopener"><img src="https://img.icons8.com/fluency/48/null/internet.png"/></a>
+</p>
+
+<p>&nbsp;</p>
+
+## **Github Stats**
```
Comment (Panquesito7) [inline Readmes/SamyabrataMaji/README.md:103]:
```suggestion
## **GitHub Stats**

```

File: Readmes/SamyabrataMaji/README.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+<img height="50" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/jest/jest-plain.svg" />&ensp;
+<img height="50" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/numpy/numpy-original.svg" />&ensp;
+<svg height="50" viewBox="0 0 128 128">
+<rect width="100%" height="100%" fill="lightblue" />
+<path style="-inkscape-stroke:none" d="M48.697 15.176h12.25v25.437h-12.25zm0 52.251h12.25v25.436h-12.25z" color="#000" fill="#130754"></path><path style="-inkscape-stroke:none" d="M48.697 48.037h12.25v12.001h-12.25z" color="#000" fill="#ffca00"></path><path style="-inkscape-stroke:none" d="M29.017 36.087h12.25v84.552h-12.25zM67.97 88.414h12.25v25.436H67.97zm0-52.297h12.25v25.437H67.97z" color="#000" fill="#130754"></path><path style="-inkscape-stroke:none" d="M67.97 68.983h12.25v12.001H67.97z" color="#000" fill="#e70488"></path><path style="-inkscape-stroke:none" d="M87.238 8.55h12.25v84.552h-12.25z" color="#000" fill="#130754"></path></svg>
+</p>
+
+###### &nbsp;
+
+### **3. Softwares:**
```
Comment (Panquesito7) [inline Readmes/SamyabrataMaji/README.md:56]: It looks better without the `:`, IMO. What do you think? 🙂
```suggestion
### **3. Softwares**
```

File: Readmes/SamyabrataMaji/README.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+<img height="50" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/androidstudio/androidstudio-original.svg" />&ensp;
+<img height="50" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/blender/blender-original.svg" />
+</p>
+
+###### &nbsp;
+
+### **4. Others:**
```
Comment (Panquesito7) [inline Readmes/SamyabrataMaji/README.md:75]:
```suggestion
### **4. Others**

```

File: Readmes/SamyabrataMaji/README.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+- 📨 How to **_reach_** me 👉 [**click here**](#contact-me)
+
+<p>&nbsp;</p>
+
+## **Tech Stack**
+
+Here's a list of all the technologies I have used. The ones I use the most are listed first.
+
+### **1. Languages:**
```
Comment (Panquesito7) [inline Readmes/SamyabrataMaji/README.md:23]:
```suggestion
### **1. Languages**

```

File: Readmes/SamyabrataMaji/README.md (hunk 2 above)
Comment (samyabrata-maji) [reply Readmes/SamyabrataMaji/README.md:57]: I think it does look better without the `:`

File: Readmes/SamyabrataMaji/README.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+<img height="50" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/typescript/typescript-original.svg" alt="TypeScript" />&ensp;
+<img height="50" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/java/java-original.svg" />&ensp;
+<img height="50" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/python/python-original.svg" />&ensp;
+<img height="50" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/kotlin/kotlin-original.svg" />&ensp;
+<img height="50" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/csharp/csharp-original.svg" />&ensp;
+</p>
+
+###### &nbsp;
+
+### **2. Frameworks and Libraries:**
```
Comment (Panquesito7) [inline Readmes/SamyabrataMaji/README.md:36]:
```suggestion
### **2. Frameworks and Libraries**
```

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻 ⚡

**Review by Greeshma2903 — APPROVED** [review]

**Comment by Greeshma2903** [comment]
Thank you for contributing! Merged 🚀

**Pull request merged**